working-groups.md

```diff
@@ -0,0 +1,168 @@
+# Tekton working group
+
+Most community activity is organized into _working groups_.
+
+Working groups follow the [contributing](./CONTRIBUTING.md) guidelines although
+each of these groups may operate a little differently depending on their needs
+and workflow.
+
+The working groups generate design docs which are kept in a
+[shared drive](https://drive.google.com/drive/folders/0AFOvPxM9MpebUk9PVA)
```

Comment:
do you think it makes sense to link to `#shared-drive` in process.md instead of duplicating this here? or move `#shared-drive` here?

Reply:
It does !